test/dynamo/test_repros.py

```diff
@@ -4765,14 +4765,27 @@ def parametrize(model: nn.Module):
                 for name, p in params:
                     if p is not None:
                         torch.nn.utils.parametrize.register_parametrization(
-                            mod, name, Parametrization(), unsafe=True
+                            mod,
+                            name,
+                            Parametrization(),
                         )
 
         m = Module()
         parametrize(m)
-        opt_m = torch.compile(m, backend="eager")
+
+        sin_found = False
+
+        def backend(gm, _):
+            nonlocal sin_found
+            for node in gm.graph.nodes:
+                if node.target is torch.sin:
+                    sin_found = True
+            return gm
+
+        opt_m = torch.compile(m, backend=backend, fullgraph=True)
         inp = torch.randn(10, 10)
         self.assertEqual(m(inp), opt_m(inp))
+        self.assertTrue(sin_found)
```

Comment:
wdyt about expanding this test by unparametrizing the module and re-running it post-compile? I'd expect a recompile since the graph that operates on the parametrized module won't apply to the unparametrized one.